tests/jballerina-integration-test/src/test/resources/auth/src/authservices/12_token_propagation_jwt_test.bal

```diff
@@ -57,12 +57,14 @@ service passthroughService12 on listener12_1 {
         path: "/"
     }
     resource function passthrough(http:Caller caller, http:Request clientRequest) {
-        var response = nyseEP12->get("/nyseStock/stocks", message = <@untainted> clientRequest);
+        var response = nyseEP12->get("/nyseStock/stocks", <@untainted> clientRequest);
         if (response is http:Response) {
             checkpanic caller->respond(response);
         } else {
+            // TODO: Remove the below casting when new lang syntax are merged.
```

Comment:
Lang syntax change will not fix this. Here we have a union type and we are trying to invoke a function on top of it, which is not allowed.

Reply:
It is already fixed with https://github.com/ballerina-platform/ballerina-lang/pull/16812

Reply:
Missed that. Thanks